pkg/actors/actors.go

```diff
@@ -1210,7 +1210,7 @@ func (a *actorsRuntime) CreateReminder(ctx context.Context, req *CreateReminderR
 	}
 
 	if a.schedulerClients != nil && a.schedulerReminders {
-		log.Info("Using Scheduler service for reminders")
+		log.Debug("Using Scheduler service for reminders")
```

Comment:
We cannot put this at info level as this will get printed millions of times are use up a lot of space & $.

If we want to keep that test, we can run daprd in debug logs for that case.